packages/tc-api-rest-handlers/__tests__/patch-rich-relationships.spec.js

```diff
@@ -0,0 +1,405 @@
+const { setupMocks, neo4jTest } = require('../../../test-helpers');
```

Comment:
Highlight #9 All tests for patching rich relationships pulled into a separate file (previously they were sprinkled about a bit unnecessarily)_